tasks/InsightAppSec/task.ts

```diff
@@ -52,12 +63,36 @@ async function run() {
         var scanId;
         var iasApi = new InsightAppSecApi(endpoint, apiKey, debugMode);
 
-        // Get the Appplication ID and Scan Config ID via API
-        var appId = await iasApi.getAppId(application);
-        console.log("Application ID for " + application + ": " + appId);
+        // Check if input is in UUID format to determine whether to get name or ID from API
```

Comment:
Could we break the logic to retrieve the missing app/scan data into their own separate functions? We can use destructuring to get the returned data: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment